src/main/java/no/sikt/sws/WorkspaceHandler.java

```diff
@@ -27,14 +30,26 @@ protected String processInput(
             RequestInfo request,
             Context context
     ) throws ApiGatewayException {
+        ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
ApiGatewayHandler has a objectMapper. I have not tried using it myself, but you should look into using it if possible.
`getObjectMapper(request)`

Reply:
I've moved the mapping to the WorkspaceResponse-class. I feel like it makes more sense that that class uses its own objectmapper instead of passing the one from ApiGateWay?